modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Outbound Response Time graphs (99, 90, 75, and 50 Percentile)::
+The response time of the 99th, 90th, 75th, and 50th percentile of outbound requests, grouped in automatic intervals.
+
+[[failures-charts]]
+== Failures
+
+The following charts appear on the *Failures* dashboard:
+
+Total Failed Inbound Requests::
+The sum of failed inbound requests in the period, grouped in automatic intervals.
+
+Total Failed Outbound Requests::
+The sum of failed outbound requests in the period, grouped in automatic intervals.
+
+Inbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
+
+Outbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
+
+Inbound - Failure Response Time by Endpoint::
+The average, maximum, and minimum response times of failed inbound requests, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Inbound - Total Failed Requests by Endpoint::
+The sum of failed inbound requests, grouped by endpoint.
+
+Outbound - Failure Response Time by HTTP endpoint::
+The average, maximum, and minimum response times of failed outbound requests, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Outbound - Total Failed Requests by HTTP Endpoint::
+The sum of failed outbound requests, grouped by endpoint.
+
+[[jvm-charts]]
+== JVM
```

Comment:
I spent some time looking up space types in JVM. I found five memory space types, but two types of heap memory: eden and survivor. I never saw "par", although I did see "per." Would you please advise me on the proper terms and capitalization for these, for the Term Presentation List?

Reply:
@jiang-wu are we sure this is correct?